--- /src/java/org/apache/commons/lang/LocaleUtils.java
+++ /src/java/org/apache/commons/lang/LocaleUtils.java
@@ -220,5 +220,5 @@ public class LocaleUtils {
      * @return true if the locale is a known locale
      */
     public static boolean isAvailableLocale(Locale locale) {
-        return cAvailableLocaleSet.contains(locale);
+        return org.apache.commons.lang.LocaleUtils.availableLocaleList().contains(locale);
     }
