--- /src/org/mockito/internal/util/MockCreationValidator.java
+++ /src/org/mockito/internal/util/MockCreationValidator.java
@@ -60,7 +60,6 @@
                 && !(Serializable.class.isAssignableFrom(classToMock))
                 && Constructors.noArgConstructorOf(classToMock) == null
                 ) {
-            new Reporter().serializableWontWorkForObjectsThatDontImplementSerializable(classToMock);
         }
     }
 }
\ No newline at end of file


--- /src/org/mockito/internal/util/Timer.java
+++ /src/org/mockito/internal/util/Timer.java
@@ -21,6 +21,5 @@
      * Starts the timer count down.
      */
     public void start() {
-        startTime = System.currentTimeMillis();
     }
 }


