--- /src/com/google/javascript/jscomp/CheckSideEffects.java
+++ /src/com/google/javascript/jscomp/CheckSideEffects.java
@@ -114,7 +114,7 @@
 					if (ancestorType == (com.google.javascript.rhino.Token.COMMA)) 
 					{ 						continue;}
 					if ((ancestorType != (com.google.javascript.rhino.Token.EXPR_RESULT)) && (ancestorType != (com.google.javascript.rhino.Token.BLOCK)))
-						return;else
+					{}else
 
 						break;
 				}