--- /src/com/google/javascript/jscomp/Compiler.java\t2019-08-11
+++ /src/com/google/javascript/jscomp/Compiler.java\t2019-08-11
@@ -252,12 +252,6 @@
           CheckLevel.OFF);
     }

-    if (options.checkGlobalThisLevel.isOn()) {
-      options.setWarningLevel(
-          DiagnosticGroups.GLOBAL_THIS,
-          options.checkGlobalThisLevel);
-    }
-
     if (options.getLanguageIn() == LanguageMode.ECMASCRIPT5_STRICT) {
       options.setWarningLevel(
           DiagnosticGroups.ES5_STRICT,


