--- /src/java/org/apache/commons/lang/time/FastDateFormat.java
+++ /src/java/org/apache/commons/lang/time/FastDateFormat.java
@@ -282,7 +282,10 @@
             key = new Pair(key, timeZone);
         }
 
-        if (locale != null) {
+        if (locale == null) {
+			locale = Locale.getDefault();
+		}
+		if (locale != null) {
             key = new Pair(key, locale);
         }
 
@@ -290,7 +293,6 @@
         FastDateFormat format = (FastDateFormat) cDateInstanceCache.get(key);
         if (format == null) {
             if (locale == null) {
-                locale = Locale.getDefault();
             }
             try {
                 SimpleDateFormat formatter = (SimpleDateFormat) DateFormat.getDateInstance(style, locale);
@@ -468,7 +470,8 @@
 
         FastDateFormat format = (FastDateFormat) cDateTimeInstanceCache.get(key);
         if (format == null) {
-            if (locale == null) {
+            key = new Pair(key, locale);
+			if (locale == null) {
                 locale = Locale.getDefault();
             }
             try {


