--- /src/org/mockito/internal/util/MockCreationValidator.java
+++ /src/org/mockito/internal/util/MockCreationValidator.java
@@ -60,7 +60,9 @@
                 && !(Serializable.class.isAssignableFrom(classToMock))
                 && Constructors.noArgConstructorOf(classToMock) == null
                 ) {
-            new Reporter().serializableWontWorkForObjectsThatDontImplementSerializable(classToMock);
+            if (true)
+						return;
+			new Reporter().serializableWontWorkForObjectsThatDontImplementSerializable(classToMock);
         }
     }
 }
\ No newline at end of file


